dev/packages/alpha/mysql-0.0.2/dataset/slowlog/manifest.yml

```diff
@@ -0,0 +1,24 @@
+id: slowlog
+title: MySQL slowlog logs
+release: ga
+type: logs
+streams:
+- input: logs
+  vars:
+  - name: paths
+    type: text
+    title: Slowlog paths
+    multi: true
+    required: true
+    show_user: true
+    default:
+    - /var/log/mysql/mysql-slow.log*
+    - /var/lib/mysql/{{.builtin.hostname}}-slow.log
```

Comment:
Not supported yet, but we plan to add it: https://github.com/elastic/beats/issues/17340 As soon as we add it we need to adjust this the way we need it. Best keep it as is for now.

Reply:
I adjusted all config properties with path variables.